agenda/domain/src/main/java/com/nibalk/tasky/agenda/domain/AgendaItem.kt

```diff
@@ -0,0 +1,65 @@
+package com.nibalk.tasky.agenda.domain
+
+import java.time.LocalDateTime
+
+sealed class AgendaItem(
+    val id: String,
+    val title: String,
+    val description: String,
+    val time: LocalDateTime,
+    val remindAt: LocalDateTime,
+    val isDone: Boolean,
```

Comment:
This is something only tasks have, so it would make sense to move this field into the `Task` class :)